src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -188,11 +186,12 @@ function MoneyRequestView({report, parentReport, parentReportActions, policyCate
                                 isLocalFile={receiptURIs.isLocalFile}
                                 transaction={transaction}
                                 enablePreviewModal
+                                canEditReceipt={canEditReceipt}
                             />
                         </View>
                     </OfflineWithFeedback>
                 )}
-                {!hasReceipt && canEditReceipt && !isSettled && canUseViolations && (
```

Comment:
We can get rid of `isSettled` here because it is already accounted for in the call to `canEditFieldOfMoneyRequest(parentReportAction, CONST.EDIT_REQUEST_FIELD.RECEIPT)` correct?

Reply:
It's already done!

Reply:
Yup!

Reply:
@joelbettner is this the change requested? Because it's already done